core/common/src/test/java/alluxio/wire/WorkerInfoTest.java

```diff
@@ -92,6 +92,7 @@ public static WorkerInfo createRandom() {
     String version = String.format("%d.%d.%d", random.nextInt(10),
         random.nextInt(20), random.nextInt(10));
     String revision = DigestUtils.sha1Hex(RandomStringUtils.random(10));
+    int numVCpu = random.nextInt(128);
```

Comment:
hmmm why is a random cpu number here pass all UTs? Like there is one `checkEquality()` you didn't update right?